news/2020-04-01-How-to-configure-wildfly-s2i-by-cli-operations.adoc

```diff
@@ -0,0 +1,446 @@
+= Configuring WildFly S2I image by using CLI Management Operations
+yborgess
+2020-04-01
+:awestruct-tags: [wildfly, galleon]
+:awestruct-layout: blog
+:source-highlighter: coderay
+:encoding: utf-8
+:lang: en
+
+== Introduction
+
+The standard and recommended way to configure the WildFly cloud images is by using environment variables. However, you could find useful for your use case to configure the server by using a custom CLI https://docs.wildfly.org/19/Admin_Guide.html#operations[management operations] script.
+
+The following post describes how you can apply management operations to configure the WildFly server image. We will show you how you can execute CLI scripts at S2I phase and how to use the extensions mechanism provided by the WildFly cloud image to execute CLI management operations at runtime.
```

Comment:
s/at S2I/at the S2I/g

Reply:
At L20 below you spell out Source-to-Image then include (S2I) and have a link. As this line is the first use of S2I in the article I think doing it that way here would be good.